src/index.js

```diff
@@ -1,35 +1,49 @@
 'use strict';
-
+//create constants
 const $tagsInput = document.getElementById('tags-input');
+const $tagsInput2 = document.getElementById('tags-input-with-default');
 const $tagsList = document.getElementById('tags-list');
-const tagsArr = ['1', 'super tag'];
+const $tagsList2 = document.getElementById('tags-list-with-default');
+const tagsArr = [];
+const tagsArr2 = ['tag1', 'some awesome tag'];
+const $modal = document.getElementById('modal-box');
+const $closeModal = document.getElementsByClassName('close-modal')[0];
 
-$tagsInput.addEventListener('keyup', ({key, target}) => {
-  if (key === 'Enter' && target.value.trim()) {
-    const $el = createTagElement(target.value);
-    const isElementExist = tagsArr.includes(target.value);
+//functions
+function eventListenerOnAdding(input, list, tags) {
+  input.addEventListener('keyup', ({key, target}) => {
+    if (key === 'Enter' && target.value.trim()) {
+      const $el = createTagElement(target.value);
+      const isElementExist = tags.includes(target.value);
 
-    if (!isElementExist) {
-      $tagsList.appendChild($el);
-      tagsArr.push(target.value);
-      target.value = '';
+      if (!isElementExist) {
+        list.appendChild($el);
+        tags.push(target.value);
+        target.value = '';
+      } else {
+        $modal.style.display = "block";
+      }
     }
-  }
-});
+  });
+}
 
-$tagsList.addEventListener('click', event => {
-  const {target} = event;
-  const isRemoveBtn = target.classList.contains('remove-btn');
-  const $foo = target.parentElement.parentElement;
-  const $tag = target.parentElement;
+$closeModal.addEventListener('click', () => {
+  $modal.style.display = "none";
```

Comment:
you must use classes for hiding/showing elements
to add class "hide" if you want to hide something or remove it if you want to show something
👎